### PR TITLE
Iconfont

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

File: themes/qgrid/source/js/menu.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 function toggleVisibility(e) {
-	e.stopImmediatePropagation();
+	var obj = (e.target ? e.target : e.srcElement);
```
Comment (klumba12) [inline themes/qgrid/source/js/menu.js]: rename it to target and can be simplified to  `e.target || e.scrElement`

File: themes/qgrid/source/js/menu.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+	if (obj.id === 'search') {
```
Comment (klumba12) [inline themes/qgrid/source/js/menu.js]: maybe add a special attribute to the element alike `stop-propagate` ?

File: themes/qgrid/source/js/menu.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
 function toggleVisibility(e) {
-	e.stopImmediatePropagation();
+	var target = e.target || e.srcElement;
+	if (target.dataset.stopPropagate) {
```
Comment (klumba12) [inline themes/qgrid/source/js/menu.js]: in ui u use data-stop-propagation and here stopPropagate, how does it work?)

Comment (etokareva) [reply themes/qgrid/source/js/menu.js]: like data-attributes))) http://htmlbook.ru/blog/atribut-data

**Review by klumba12 — APPROVED** [review]

**Pull request merged**